flexmeasures/data/models/reporting/pandas_reporter.py

```diff
@@ -0,0 +1,114 @@
+import pandas as pd
+from flexmeasures.data.models.reporting import Reporter
+from flexmeasures.data.schemas.reporting.pandas_reporter import (
+    PandasReporterConfigSchema,
+)
+
+
+class PandasReporter(Reporter):
+    """This reporter applies a series of pandas methods on"""
+
+    __version__ = "1"
+    __author__ = None
+    schema = PandasReporterConfigSchema()
+
+    def deserialize_report_config(self):
+        # call super class deserialize_report_config
+        super().deserialize_report_config()
+
+        # extract PandasReporter specific fields
+        self.transformations = self.reporter_config.get("transformations")
+        self.final_df_output = self.reporter_config.get("final_df_output")
+
+    def _compute(self) -> pd.Series:
+        """"""
+        # apply pandas transformations to the dataframes in `self.data``
+        self.apply_transformations()
+
+        final_output = self.data[self.final_df_output]
+        return final_output
+
+    def get_object_or_literal(self, value, method):
+        """This method allows using the dataframes as inputs of the Pandas methods that
+        are run in the transformations. Make sure that they have been created before accessed.
+
+        This works by putting the symbol `@` in front of the name of the dataframe that we want to reference.
+        For instance, to reference the dataframe test_df, which lives in self.data, we would do `@test_df`.
+
+        This functionality is disabled for methods `eval`and `query` to avoid interfering their internal behaviour
+        given that they also use `@` to allow using local variables.
+
+        Examples
+        >> self.get_object_or_literal(["@sensor_1", "@sensor_2"], "sum")
+        [[                    ...n: 0:00:00,                     ...n: 0:00:00]]
+        """
+
+        if method in ["eval", "query"]:
+            return value
```

Comment:
I've added the warning. In regards of raising a `ValueError`, it happens that `eval` and `query` use the symbol `@`, as well, and we don't want to interfere with that.

Reply:
Ah I get it now. In that case, we might reduce the warning level to "debug". In case someone is intentionally using eval or query in their reporter config, we *do* want to support that, and then I'd prefer to not overflow their logs with warnings.